src/Common/AbstractMailer.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace Omnimail\Common;
+
+use Omnimail\EmailSenderInterface;
+
+abstract class AbstractMailer implements EmailSenderInterface
+{
```

Comment:
The methods here were not used and used `symfony/http-foundation` so I removed them for now. If we re-add them, it would be best to avoid `symfony/http-foundation`.